inst/sql/sql_server/checks/concept_counts_by_vocabulary.sql

```diff
@@ -3,12 +3,12 @@
 select vocabulary.vocabulary_id                                  as id,
        vocabulary.vocabulary_name                                as name,
        vocabulary.vocabulary_version                             as version,
-       sum(case standard_concept when 'S' then 1 else 0 end)     as num_standard,
-       sum(case standard_concept when 'C' then 1 else 0 end)     as num_classification,
-       sum(case when standard_concept = '' OR standard_concept IS NULL  then 1 else 0 end) as num_non_standard
+       sum(case standard_concept when 'S' then 1 else 0 end)     as S,
+       sum(case standard_concept when 'C' then 1 else 0 end)     as C,
+       sum(case when standard_concept = '' OR standard_concept IS NULL  then 1 else 0 end) as "-"
```

Comment:
Might be a bit unclear what - means, maybe we should add a footnote?

Reply:
Yes, description of S, C and - is now included in the caption (screenshot did not include that indeed)